iam.tf

```diff
@@ -1,7 +1,7 @@
 # Create a role that allows the instance to read its certs from S3.
 
 module "certreadrole" {
-  source = "github.com/cisagov/cert-read-role-tf-module"
+  source = "github.com/cisagov/cert-read-role-tf-module?ref=test_tf_aws_provider_v3"
```

Comment:
Don't forget to revert this change before merging.

Reply:
From the **Motivation and Context** section above 😄 :
> I will keep this PR as a draft until cisagov/cert-read-role-tf-module#19 is merged. After that, I will revert 8041de9, finalize this PR, and request review.

Reply:
Reverted with https://github.com/cisagov/openvpn-server-tf-module/pull/48/commits/301c3edd590d9a9e1581da804f4ddc0de5f6e48e.